ci/Dockerfile.buster.amd64

```diff
@@ -21,7 +20,7 @@ RUN apt-get update ;\
 
 RUN apt-get update ;\
   apt-get -y dist-upgrade --auto-remove --purge ;\
-  apt-get -y install python3-pip wget build-essential git iw locales wpasupplicant;\
```

Comment:
I assume this is install in the install script, right?

Reply:
yeah - imho it should at least be. So we can be sure it is available. But if we install it in our dockerimage we might some day remove that line from the install script and all hell breaks loose again.